### PR TITLE
Update payments-api-overview.md

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-2956](https://jira.bigcommerce.com/browse/DEVDOCS-2956)

## What changed?
Added the Capture API and Refunds API options

### CODE REVIEW

File: docs/api-docs/payments/payments-api-overview.md [hunk 1]
```diff
@@ -619,11 +619,11 @@ The Payments API does not support adding a third-party gateway. Payments are pro
 
 **Can I issue a refund?**
 
-Refunds can be issued either using the [Control Panel](https://support.bigcommerce.com/s/article/Processing-Refunds) or through the payment gateway directly.
+You can issue a refund using the [Control Panel](https://support.bigcommerce.com/s/article/Processing-Refunds), the Refunds API, or through the payment gateway directly.
```
Comment (ektravel) [inline docs/api-docs/payments/payments-api-overview.md:622]:
```suggestion
You can issue a refund using the [control panel](https://support.bigcommerce.com/s/article/Processing-Refunds), the Refunds API, or through the payment gateway directly.
```

File: docs/api-docs/payments/payments-api-overview.md [hunk 2]
```diff
@@ -619,11 +619,11 @@ The Payments API does not support adding a third-party gateway. Payments are pro
...
 
 **How do I process payment for a capture credit card?**
 
-Once a payment has been authorized, the capture step will need to be completed using the [Control Panel](https://support.bigcommerce.com/s/article/How-can-I-set-my-payment-gateway-to-only-authorize-transactions-and-not-capture-the-funds-automatically).
+Once you have an authorized payment, perform the capture step using the [Control Panel](https://support.bigcommerce.com/s/article/How-can-I-set-my-payment-gateway-to-only-authorize-transactions-and-not-capture-the-funds-automatically) or the Capture API.
```
Comment (ektravel) [inline docs/api-docs/payments/payments-api-overview.md:626]:
```suggestion
Once you have an authorized payment, perform the capture step using the [control panel](https://support.bigcommerce.com/s/article/How-can-I-set-my-payment-gateway-to-only-authorize-transactions-and-not-capture-the-funds-automatically) or the Capture API.
```

File: docs/api-docs/payments/payments-api-overview.md [hunk 3]
```diff
@@ -557,7 +557,7 @@ The `payment_access_token` is not from the payment provider. It is created by Bi
 A declined payment will return a 4XX error with details if available.
 
 ### Authorization
-If you configure a payment gateway for authorization only, authorization happens at the time of processing. You will need to capture the order later through the control panel. If you configure a payment gateway for authorization and capture, the payment will be authorized and captured at the time of processing.
+If you configure a payment gateway for authorization only, authorization happens at the time of processing. You will need to capture the order later through the control panel or via the Capture API. If you configure a payment gateway for authorization and capture, the payment will be authorized and captured at the time of processing.
```
Comment (ektravel) [inline docs/api-docs/payments/payments-api-overview.md:560]:
```suggestion
If you configure a payment gateway for authorization only, authorization happens at the time of processing. You will need to capture the order later using the control panel or the Capture API. If you configure a payment gateway for authorization and capture, the payment will be authorized and captured at the time of processing.
```

**Review by ektravel — COMMENTED** [review]
Added a few suggestions

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**